app/src/main/res/layout/fragment_about.xml

```diff
@@ -3,27 +3,27 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="fill_parent"
-    android:layout_height="fill_parent">
+            xmlns:app="http://schemas.android.com/apk/res-auto"
+            xmlns:tools="http://schemas.android.com/tools"
+            android:layout_width="fill_parent"
+            android:layout_height="fill_parent"
+            tools:context="org.mozilla.fenix.settings.AboutFragment">
 
     <androidx.constraintlayout.widget.ConstraintLayout
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:app="http://schemas.android.com/apk/res-auto" xmlns:tools="http://schemas.android.com/tools"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        tools:context="org.mozilla.fenix.settings.AboutFragment">
+        android:layout_height="wrap_content">
 
         <ImageView
             android:id="@+id/wordmark"
-            android:importantForAccessibility="no"
```

Comment:
Should we read a content description for our logo?

Reply:
"About Firefox Preview" is written below the logo, so I thought that a content description isn't needed here.